apps/web/components/AddCard/CardForm.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Stack, Group, Title, Text, Flex, ScrollArea } from '@mantine/core';
+import Link from 'next/link';
+import { Button } from '@ui/button';
+import { PaymentMethods } from './PaymentMethods';
+import { AddNewPaymentMethodForm } from './PaymentMethods/AddNewPaymentMethodForm';
+import { colors } from '@config';
+import { ICardData } from '@impler/shared';
+import { CurrentCardSection } from './CurrentCardSection';
+import { useStyles } from './CardForm.styles';
+
```

Comment:
@thisismayuresh Please sort it properly.

Reply:
yeah sorted the import statements.